app/routes/pages/components/PageEditor.tsx

```diff
@@ -53,11 +55,11 @@ type FormValues = Omit<ApiRequestBody, 'category'> & {
 const TypedLegoForm = LegoFinalForm<FormValues>;
 
 const PageEditor = () => {
-  const { pageSlug } = useParams<{ pageSlug: string }>();
+  const { pageSlug } = useParams<PageDetailParams>() as PageDetailParams;
   const page = useAppSelector((state) =>
-    selectPageBySlug(state, {
-      pageSlug,
-    }),
+    pageSlug
+      ? (selectPageBySlug(state, pageSlug) as AuthDetailedPage)
+      : undefined,
```

Comment:
Is this not handled by `selectPageBySlug`?

Reply:
It is after rebasing😁